source/elements/oneMKL/source/domains/blas/gemm_ext.rst

```diff
@@ -69,7 +73,7 @@ gemm_ext
       .. container:: dlsyntaxpara
 
 
-         .. cpp:function::  void gemm_ext(queue &exec_queue, transpose         transa, transpose transb, offset offset_type, std::int64_t m,         std::int64_t n, std::int64_t k, Ts alpha, buffer<Ta,1> &a,         std::int64_t lda, Ta ao, buffer<Tb,1> &b, std::int64_t ldb, Tb         bo, Ts beta, buffer<Tc,1> &c, std::int64_t ldc, buffer<Tc,1>         &co)
+         .. cpp:function::  void onemkl::blas::gemm_ext(sycl::queue &queue, transpose         transa, transpose transb, offset offset_type, std::int64_t m,         std::int64_t n, std::int64_t k, Ts alpha, sycl::buffer<Ta,1> &a,         std::int64_t lda, Ta ao, sycl::buffer<Tb,1> &b, std::int64_t ldb, Tb         bo, Ts beta, sycl::buffer<Tc,1> &c, std::int64_t ldc, sycl::buffer<Tc,1>         &co)
```

Comment:
I have not looked at the entire document, but you might want to have a section about namespaces. You could say that queue is always a sycl::queue so you won't have to put sycl:: everywhere. And it is probably better not to have onemkl:: namespace, but document that everything is in that namespace.

Reply:
our thought was to make the individual api's page definition copyable as best we can and as specific as we can without assumption of "using xxxxx" having been present.  We decided to add all of these namespaces in, in this one place.  Anywhere else it is referenced, it would not have all these details.  There is also a section in the Architecture chapter on namespaces.